### PR TITLE
Command Line support for Model Archive

### PR DESCRIPTION
**Pull request by anchen1011**
With this portion of code, customers will be able to import a model archive into Lambda Function with one line of command before deploy it.

### CODE REVIEW

File: lit-cli/client.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+        do_install('mxnet', requirement=False, target='.')
+    # remove temp path
+    shutil.rmtree(dirpath)
+    if not is_url:
+        # upload model_archive
+        status = subprocess.call("aws s3 cp " + model_archive + " s3://" + model_bucket + " --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers", shell=True)
+        if status != 0:
+            click.echo("Failed to upload model archive to S3. Interrupt.")
+            return
+        url_mar = "https://" + model_bucket + '.s3.amazonaws.com/' + model_archive.split('/')[-1]
```
Comment (vrakesh) [inline lit-cli/client.py]: use string join

File: lit-cli/client.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+        do_install(os.path.join(dirpath, 'requirements.txt'), requirements=True, target='.')
+    if check_existence('mxnet/', dirpath) == False and check_existence('mxnet/', '.') == False:
+        # rollback numpy to 1.13 (avoid problems with the latest version)
+        do_install('numpy==1.13.3', requirement=False, target='.')
+        do_install('mxnet', requirement=False, target='.')
+    # remove temp path
+    shutil.rmtree(dirpath)
+    if not is_url:
+        # upload model_archive
+        status = subprocess.call("aws s3 cp " + model_archive + " s3://" + model_bucket + " --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers", shell=True)
```
Comment (vrakesh) [inline lit-cli/client.py]: string join here as well

File: lit-cli/client.py (hunk 1 above)
Comment (anchen1011) [reply lit-cli/client.py]: Modified accordingly

File: lit-cli/client.py (hunk 2 above)
Comment (anchen1011) [reply lit-cli/client.py]: Modified accordingly

**Comment by nswamy** [comment]
Closing this PR since we agreed to use the SAM experience.

**Pull request closed without merging**